docs/DeveloperGuide.md

```diff
@@ -451,6 +457,193 @@ Given below are instructions to test the app manually.
+i. Test case: `list /u`  
+Expected: All existing undone tasks in the tasklist will be printed out.  
+
+2.5.1.4. Listing all urgent existing tasks in the tasklist.  
+i. Test case: `list /urgent`  
+Expected: All existing urgent tasks in the tasklist will be printed out.  
+
+#### 2.6 Executing a mode switch
+
+2.6.1. Switching between Timetable mode and Notebook mode.  
+i. Prerequistes: User must be in the Timetable mode. Enter `mode /t` command to enter Timetable mode.  
+ii. Test case: `mode /n`   
+Expected: Mode is switched to Notebook mode.  
+iii. Test case: `mode /t`  
+Expected: Mode is switched to Timetable mode.  
+iv. Test case: `mode`  
+Expected: An error message along with a formatting guideline message will be printed in the command line interface..  
+
+#### 2.7 Finding a Task using a keyword
+
+2.7.1. Finding all tasks that contains a specified keyword.  
+i. Prerequistes: User must be in the Timetable mode. Enter `mode /t` command to enter Timetable mode. List all tasks in the tasklist using the `list` command. There must be existing tasks in the list.  
+ii. Test case: `Find Project`  
+Expected: All tasks with tags that contains the word 'Project' will be printed. If the tasks do not have tags, it will list the tasks with titles that contain the keyword 'Project'.  
+iii. Test case: `Find`  
+Expected: An error message along with a formatting guideline message will be printed in the command line interface.  
+
+#### 2.8 Adding a Notebook/Section/Page
+
+2.8.1. Adding a respective notebook/section/page in their respective list.  
+i. Prerequistes: User must be in the Notebook mode. Enter `mode /n` command to enter Notebook mode.    
+
+2.8.1.1. Adding a Notebook.  
+i. Prerequistes: User must not be in a selected notebook or section. User should type `mode /n` before executing this test case.   
+ii. Test case: `add /nCS2113T`  
+Expected: Notebook 'CS2113T' is added to the notebookShelf with a success message printed.  
+
+2.8.1.2. Adding a Section.  
+i. Prerequistes: User must select a notebook. Enter `select /s[NOTEBOOK]` to enter a specified notebook.    
+ii. Test case: `add /nChapter 1`  
+Expected: Section 'Chapter 1' is added to the specified notebook with a success message printed.  
+
+2.8.1.3. Adding a Page.  
+i. Prerequistes: User must select a Section. Enter `select /s[SECTION]` to enter a specified section.    
+ii. Test case: `add /p Page 1 ; a test page`  
+Expected: Page 'Page 1' along with content 'a test page' is added to the specified section with a success message printed.  
+
+#### 2.9 Selecting a Notebook/Section/Page
+
+2.9.1. Selecting a respective notebook/section/page in their respective list.  
+i. Prerequistes: User must be in the Notebook mode. Enter `mode /n` command to enter Notebook mode.  
+
+2.9.1.1. Selecting a Notebook.  
+i. Prerequistes: User must not be in a section. User should type `mode /n` before executing this test case.   
```

Comment:
nope, you can select a notebook anywhere.

Reply:
also note sure if you noticed but it's prerequisites... HAHAHAH I think you just copied paste for all the prerequisites part so it's just prerequistes..hehe

Reply:
Ohh okok will change it to just staying the notebook mode then